### ISSUE TITLE
examples in docs are wrong

### ISSUE THREAD

**Issue by pYr0x** [swiftmailer/swiftmailer#925]
by changing to swiftmailer 6 the creation of a mailer instance is now by creating a new class instead of calling a static method.
in some examples it seems that only the static calling is replaced by `new`.

http://swiftmailer.org/docs/introduction.html
*basic usage*
```
require_once '/path/to/vendor/autoload.php';

// Create the Transport
$transport = (new Swift_SmtpTransport('smtp.example.org', 25))
  ->setUsername('your username')
  ->setPassword('your password')
;

// Create the Mailer using your created Transport
$mailer = new Swift_Mailer($transport);

// Create a message
$message = new Swift_Message('Wonderful Subject')
  ->setFrom(['john@doe.com' => 'John Doe'])
  ->setTo(['receiver@domain.org', 'other@domain.org' => 'A name'])
  ->setBody('Here is the message itself')
  ;

// Send the message
$result = $mailer->send($message);
```
should be
```
require_once '/path/to/vendor/autoload.php';

// Create the Transport
$transport = (new Swift_SmtpTransport('smtp.example.org', 25))
  ->setUsername('your username')
  ->setPassword('your password')
;

// Create the Mailer using your created Transport
$mailer = new Swift_Mailer($transport);

// Create a message
$message = (new Swift_Message('Wonderful Subject'))
  ->setFrom(['john@doe.com' => 'John Doe'])
  ->setTo(['receiver@domain.org', 'other@domain.org' => 'A name'])
  ->setBody('Here is the message itself')
  ;

// Send the message
$result = $mailer->send($message);
```

this missing `(` `)` are in different examples around the docs.
http://swiftmailer.org/docs/sending.html#smtp-with-a-username-and-password

**Comment by Rotzbua** [comment]
Pls make a pull request to:
https://github.com/swiftmailer/swiftmailer/blob/master/doc/introduction.rst

**Comment by chshanovskiy** [comment]
I've created this one https://github.com/swiftmailer/swiftmailer/pull/929

**Comment by c960657** [comment]
Fixed in #929.
